### ISSUE TITLE
Feature Request / Multiple py files

### ISSUE THREAD

**Issue by redfood** [AndersonBY/p5py#1]
Thank you for creating p5py.  It is almost exactly what I was looking for!  

I'm not sure if you are interested in feature requests but... It would be great if p5py could support multiple .py files.  

So, for example, in your sample project.  The `Wave` class could be defined in `wave.py` and then in the main sketch file, you could `import wave`.  (I imagine each `.py` file would be in a tab like in the Processing SDK)

**Comment by AndersonBY** [comment]
Yeah I think it would be great if we can make multiple .py files. But I'm not very familiar with front-end development. So I may need to spend some time to see how to make it.

**Comment by redfood** [comment]
I just whipped up a minimal example of tabs.  https://github.com/redfood/minimalTabDemo

Hope it helps!
